apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/Program.cs

```diff
@@ -0,0 +1,286 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.MSBuild;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+using static Google.Cloud.Bigtable.V2.GenerateClient.RoslynHelpers;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Google.Cloud.Bigtable.V2.GenerateClient
+{
+    class Program
+    {
+        private const string AppProfileIdFieldName = "_appProfileId";
+        private const string AppProfileIdPropertyName = "AppProfileId";
+        private const string GetClientMethodName = "GetClient";
+
+        private static async Task<int> Main(string[] args)
+        {
+            // TODO: Figure out why `dotnet run` from generateapis.sh is sending 6 args instead of 3 as in: arg1 arg2 arg3 arg1 arg2 arg3
```

Comment:
You don't have a "--" in generateapis.sh to separate the program arguments from the other dotnet arguments. If you change it to 

```sh
dotnet run -f net461 -p apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient -- \
apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj \
BigtableServiceApiClient \
BigtableClient
```

then that might fix it. (I don't know why it would be doubling up, but...)

Reply:
I tried that too. Same result. From the docs, it looked like it was optional for this purpose anyway.

Reply:
Okay, that's weird. I'll have a look at it at some point, but it does seem very strange...